### PR TITLE
seil.rb: Upgrade to 11.3.0

### PR DESCRIPTION
**Pull request by xight**

### CODE REVIEW

File: Casks/seil.rb [hunk 1]
```diff
@@ -4,12 +4,13 @@
 
   url "https://pqrs.org/osx/karabiner/files/Seil-#{version}.dmg"
   appcast 'https://pqrs.org/osx/karabiner/files/seil-appcast.xml',
-          :sha256 => '1491ab1babc8ec5c65d845f9b0bf68b9ef575cc46b1ec7e41696b4eeb3fa3e7e'
+          :sha256 => '4c31a2e65806980f97638ce979d8a7654fef3664af921c0ae43eb5fbf5f418b5'
+
```
Comment (vitorgalvao) [inline Casks/seil.rb]: This empty line must go.

File: Casks/seil.rb [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
   name 'Seil'
   homepage 'https://pqrs.org/osx/karabiner/seil.html'
   license :public_domain
 
-  pkg 'Seil.pkg'
+  pkg 'Seil.sparkle_guided.pkg'
```
Comment (vitorgalvao) [inline Casks/seil.rb]: Where are you getting this from?

Comment (xight) [inline Casks/seil.rb]: I got the file from this URL.
https://pqrs.org/osx/karabiner/seil.html.ja

I found that pkg name was changed. so I fixed a pkg name.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in bea7d3fa83208506785504a0ce03b1e2930293f7. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**